README.md

```diff
@@ -2,33 +2,24 @@
 
 This repository contains the Thrift schema required for displaying story packages.
 
-#Downloading
+## Downloading
 
 To download you should be able to simply add a dependency similar to the following (with your desired version):
 
 `"com.gu" %% "story-packages-model" % "x.y.z"`
 
-# Publishing
-## You will need a Bintray account!
-In order to publish a new version of this schema you'll need a Bintray account.
-
-1. Go to `bintray.com` and login with your GitHub account.
-2. Someone will need to invite you to The Guardian Bintray org. Ask super nicely and they just might do it. They should also make you an admin.
-3. You will need an API key.
-  1. Go to your profile
-  2. Click the Edit button near your profile name (top left).
-  3. At the bottom of the list on the left will be the API key section containing your key. Keep this key handy for the next step.
-4. Setup your username/API key locally.
-  1. In this project run `sbt bintrayChangeCredentials`
-  2. Enter your username and API key as prompted.
-  3. This will save your creds locally and you shouldn't need to change them unless you refresh your API key.
-
-## How to publish a new version
-So you've made some edits and you want to publish a new version of this schema as a package to Bintray jcenter...
-
-1. Make your changes
-2. Bump the version in `version.sbt` (Remember to follow the SemVer guidelines)
-3. Ensure the project still builds.
-4. Run `sbt '+ publish'`
-5. If you setup your Bintray account correctly then this should publish your new version to Bintray!
-6. If for some reason you want to remove your package from Bintray you can run `sbt bintrayUnpublish` which will remove the package *at the current version*.
\ No newline at end of file
+## How to release
+
+Ensure you have the following installed on your machine:
+ - `tsc` (`brew install typescript`)
+ - `npm` (not sure! there are so many ways to install it)
+ 
+Ensure you have an NPM account, part of the [@guardian](https://www.npmjs.com/org/guardian) org with a [configured token](https://docs.npmjs.com/creating-and-viewing-authentication-tokens)
+
+You'll also need a sonatype account to publish the library to Maven.
+
+```sbtshell
+release // will release the scala / thrift projects
```

Comment:
given this advice, do we need to specifiy `releaseCrossBuild := true` ?

Reply:
You're probably right!